CarlosDahmen/setGame
# Set 02/set found

**What does this PR do?**
Once three cards are selected, it applies an overlay over the cards (green if the cards are a set, red if the cards are not a set), and after one second, cleans up the deck as necessary (removes selected cards and deals new ones if they were a set, resets selected cards if not), and updates the score accordingly.

**Any background context you want to provide?**

**Screenshots (if appropriate):**

**How to verify it?**
Run the app, select three cards that are not a set, once the third cards is selected, a red overlay will appear over them. After 1 second, the overlay will be removed and the score will decrease by 1. 
Then, select three cards that are a set, a green overlay will appear over them for 1 second. After one second, the selected cards will be removed, new cards will be dealt in their place, and the score will increase by one.
